.github/workflows/addToProject.yml

```diff
@@ -0,0 +1,30 @@
+
+name: Auto Assign to Project(s)
+
+on:
+  issues:
+    types: [opened, edited, milestoned]
+  pull_request:
+    types: [opened, edited, milestoned]
+env:
+  MY_GITHUB_TOKEN: ${{ secrets.APM_TECH_USER_TOKEN }}
```

Comment:
Something seems to be not ok with this part - [it says](https://github.com/elastic/apm-agent-dotnet/pull/1143/checks?check_run_id=1742808668) 

```
MY_GITHUB_TOKEN not defined
```

I wonder if this is needed here.

Reply:
I guess this is because this execution was performed on my repo (origin of the PR) and my repo does not have the APM_TECH_USER_TOKEN configured.

I actually tested the exact same workflow file already with the java agent, there it worked